
int8  no
int8  pos
